web/jobs/jobs.go

```diff
@@ -225,6 +241,7 @@ func getAllTriggers(c echo.Context) error {
 	if err != nil {
 		return wrapJobsError(err)
 	}
+	// TODO: we could potentially benefit from an index on 'worker_type' field.
```

Comment:
In theory, yes. In practice, if we have so many triggers that this loop is slow, we will have so more issues that we don't really care about this particular performance issue.